ffmpegdecklink.rb

```diff
@@ -105,6 +106,7 @@ def install
     args << "--enable-libfontconfig" if build.with? "fontconfig"
     args << "--enable-libgme" if build.with? "game-music-emu"
     args << "--enable-libgsm" if build.with? "libgsm"
+    args << "--enable-libiec61883" if build.with? "iec61883"
```

Comment:
can this `if` also test the OS?

Reply:
If you want it! I was originally going to write it like that, but then I figured that anyone adding that flag would have gotten it out of the Linux install docs, so maybe redundant. I'll add it in though - makes sense I suppose!

Reply:
I presume (untested) that
```rb
args << "--enable-libiec61883" if (build.with? "iec61883") && OS.linux?
```
would work.

Reply:
Does indeed!